cfme/tests/webui/test_advanced_search.py

```diff
@@ -16,108 +21,15 @@
-    Param(ConfigSystem, 'All', 'configuration_management_systems',
-     'Configured System (Red Hat Satellite) : Hostname',
-     ('sidebar.configured_systems', "All Configured Systems")),
-
-    Param('network_providers', 'All', 'network_managers', 'Network Manager : Name', None),
-    Param('cloud_networks', 'All', 'network_networks', 'Cloud Network : Name', None),
-    Param('network_subnets', 'All', 'network_subnets', 'Cloud Subnet : Name', None),
-    Param('network_routers', 'All', 'network_routers', 'Network Router : Name', None),
-    Param('network_security_groups', 'All', 'network_security_groups', 'Security Group : Name',
-          None),
-    Param('network_floating_ips', 'All', 'network_floating_ips', 'Floating IP : Address', None),
-    Param('network_ports', 'All', 'network_ports', 'Network Port : Name', None),
-    Param('balancers', 'All', 'network_load_balancers', 'Load Balancer : Name', None),
-
-    Param('volumes', 'All', 'block_store_volumes', 'Cloud Volume : Name', None),
-    Param('volume_snapshots', 'All', 'block_store_snapshots', 'Cloud Volume Snapshot : Name', None),
-    Param('volume_backups', 'All', 'block_store_backups', 'Cloud Volume Backup : Name', None),
-
-    Param('object_store_containers', 'All', 'object_store_containers',
-     'Cloud Object Store Container : Name', None),
-    Param('object_store_objects', 'All', 'object_store_objects',
-     'Cloud Object Store Object : Name', None),
-
-    Param('ansible_tower_providers', 'All', 'ansible_tower_explorer_provider',
-     'Automation Manager (Ansible Tower) : Name',
-     ('sidebar.providers', 'All Ansible Tower Providers')),
-    Param('ansible_tower_systems', 'All', 'ansible_tower_explorer_system',
-     'Configured System (Ansible Tower) : Hostname',
-     ('sidebar.configured_systems', 'All Ansible Tower Configured Systems')),
-    Param('ansible_tower_job_templates', 'All', 'ansible_tower_explorer_job_templates',
-     'Template (Ansible Tower) : Name', ('sidebar.job_templates', 'All Ansible Tower Templates')),
-    Param('ansible_tower_job_templates', 'All', 'ansible_tower_explorer_job_templates',
-     'Job Template (Ansible Tower) : Name',
-     ('sidebar.job_templates', 'All Ansible Tower Job Templates')),
-
-    Param('ansible_tower_jobs', 'All', 'ansible_tower_jobs', 'Ansible Tower Job : Name', None)
-]
-
 pytestmark = [
-    test_requirements.filtering,
-    pytest.mark.parametrize(
-        'param', params_values,
-        ids=['{}-{}'.format(param.entity, param.destination.lower()) for param in params_values]
-    ),
     pytest.mark.uncollectif(
         lambda param, appliance: (
-            appliance.version > '5.10' and
             (param.collection in (ConfigManager, 'ansible_tower_providers') or
-            param.filter == 'Job Template (Ansible Tower) : Name')
+            param.filter == 'Job Template (Ansible Tower) : Name') or
+            (appliance.version >= '5.11' and param.entity == 'network_load_balancers')
```

Comment:
would be nice to add some small comment with the RFE number, that load balancers are no longer supported in 5.11